stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/chat/navigation/GalleryImageAttachmentDestination.kt

```diff
@@ -25,10 +25,9 @@ public class GalleryImageAttachmentDestination(
             return
         }
 
-        val userName = message.user.name
         val createdAt = message.getCreatedAtOrThrow().time
         val attachmentIndex = message.attachments.indexOf(attachment)
 
-        start(AttachmentGalleryActivity.createIntent(context, userName, createdAt, attachmentIndex, imageUrls))
+        start(AttachmentGalleryActivity.createIntent(context, createdAt, attachmentIndex, message))
```

Comment:
I see that previously we did filtering to ensure that we're going to show only images. I think we have to pass to `createIntent` also filtered list of attachments